v3/how-to-guides/02-pallet-design/a-add-contracts-pallet/index.mdx

```diff
@@ -88,110 +116,196 @@ Implement the Contracts pallet [configuration traits][contracts-config-rustdocs]
-./target/release/node-template --dev --tmp
+./target/release/node-template --dev
 ```
 
 ## Examples
 
-- `canvas-node` [runtime](https://github.com/paritytech/canvas-node/blob/master/runtime/src/lib.rs#L361)
-- `canvas-node` [rpc file](https://github.com/paritytech/canvas-node/blob/master/node/src/rpc.rs)
+These are a couple of repositories that contain node configurations with
+the Contracts pallet. In case you want to look up how something can
+be configured in specific scenarios these node configurations can be
+helpful.
+
+### Developer Node
+
+[`substrate-contracts-node`][substrate-contracts-node] is a standalone node,
+based on Substrate's [`node-template`](https://github.com/paritytech/substrate/tree/master/bin/node-template).
+The node was configured to include the Contracts pallet.
+It is tracking Substrate's `master` branch.
+
+This particular node contains a couple of modifications that make it unsuitable for a
+production deployment, but a great fit for development and testing.
+The modifications are described in the readme.
+
+### Parachain Node
+
+The [`contracts-parachain`][contracts-parachain] node has been configured to be
+a parachain for Rococo, a testnet for Polkadot and Kusama parachains.
+It has been configured as a common good parachain, as such it uses the Rococo relay
+chain's native token `ROC` instead of defining a token of its own.
+
+### Default Node from Substrate repository
+
+The Substrate repository contains a [node configuration][bin-runtime-contracts-config]
+that includes all pallets shipped with Substrate.
 
 ## Related material
 
-- [`pallet_contracts` crate](/rustdocs/latest/pallet_contracts/index.html)
-- [`pallet_contracts_rpc` crate](/rustdocs/latest/pallet_contracts_rpc/index.html)
+- The [`pallet_contracts` crate documentation](/rustdocs/latest/pallet_contracts/index.html).
+- The [`pallet_contracts_rpc` crate documentation](/rustdocs/latest/pallet_contracts_rpc/index.html).
+- [Develop Smart Contracts with ink!][ink-tutorial] ‒ A tutorial on how to write smart contracts for the Contracts pallet.
+
 
 [create-first-chain-tutorial]: /tutorials/v3/create-your-first-substrate-chain
-[contracts-config-rustdocs]: /rustdocs/latest/pallet_contracts/trait.Config.html
-[contracts-frame-gh]: https://github.com/paritytech/substrate/blob/master/frame/contracts/src/lib.rs#L144
-[bin-runtime-contracts-frame]: https://github.com/paritytech/substrate/blob/632b32300eb9376767c2ae7b38e79b3f7f5329b1/bin/node/runtime/src/lib.rs#L887-L903
-[contracts-api-rustdocs]: /rustdocs/latest/pallet_contracts_rpc_runtime_api/trait.ContractsApi.html
+[contracts-config-rustdocs]: /docs.substrate.io/rustdocs/latest/pallet_contracts/pallet/trait.Config.html
+[contracts-frame-gh]: https://github.com/paritytech/substrate/blob/master/frame/contracts/src/lib.rs
+[bin-runtime-contracts-frame]: https://github.com/paritytech/substrate/blob/b75a253f148aa36fa17cf795b9f2fc2f22d0fcc5/bin/node/runtime/src/lib.rs#L1078-L1095
```

Comment:
Should all links be to a specific tag? Ideally based on a concrete release of the tooling and ink. So we can reason about things easier and maintain on releases